go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/yaml
 
-go 1.12
+go 1.21
```

Comment:
Also updated this to the lowest version which gets tested

Reply:
revert this bit... we should make this reflect the actual minimum required go version, and I don't *think* we require newer versions than this

Reply:
Makes sense, removed